resmo/manifest.json

```diff
@@ -0,0 +1,64 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "0439ab8c-a8f5-4b68-88c1-7083eed72e0d",
+  "app_id": "resmo",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Integrate with Resmo to visualize and secure your Datadog resources.",
+    "title": "Resmo",
+    "media": [
+      {
+        "media_type": "image",
```

Comment:
Note that any images that do not have a 16:9 aspect ratio will be cut off. We require all images to fit into this aspect ratio. Please see our [media guidelines here](https://docs.datadoghq.com/developers/integrations/create_a_tile/?tab=buildatileontheintegrationspage#media-carousel).